mhapsite/mhap/models.py

```diff
@@ -45,6 +45,7 @@ def get_list_url(self):
 #http://stackoverflow.com/questions/1737017/django-auto-now-and-auto-now-add
 class ChatMessages(models.Model):
     message = models.CharField(max_length=255)
+    message_type = models.CharField(max_length=255, default="default")
```

Comment:
Okay, I guess what I'm wondering then is: why is this helpful? Just for organization?